extensions/kafka/src/main/java/io/deephaven/kafka/IgnoreImpl.java

```diff
@@ -3,22 +3,85 @@
  */
 package io.deephaven.kafka;
 
+import io.confluent.kafka.schemaregistry.SchemaProvider;
+import io.confluent.kafka.schemaregistry.client.SchemaRegistryClient;
+import io.deephaven.engine.table.ColumnDefinition;
+import io.deephaven.engine.table.Table;
+import io.deephaven.engine.table.TableDefinition;
 import io.deephaven.kafka.KafkaTools.Consume;
-import io.deephaven.kafka.KafkaTools.DataFormat;
-import io.deephaven.kafka.KafkaTools.Produce.KeyOrValueSpec;
+import io.deephaven.kafka.KafkaTools.KeyOrValue;
+import io.deephaven.kafka.KafkaTools.KeyOrValueIngestData;
+import io.deephaven.kafka.KafkaTools.Produce;
+import io.deephaven.kafka.ingest.KeyOrValueProcessor;
+import io.deephaven.kafka.publish.KeyOrValueSerializer;
+import org.apache.commons.lang3.mutable.MutableInt;
+import org.apache.kafka.common.serialization.ByteBufferDeserializer;
+import org.apache.kafka.common.serialization.ByteBufferSerializer;
+import org.apache.kafka.common.serialization.Deserializer;
+import org.apache.kafka.common.serialization.Serializer;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
 
 class IgnoreImpl {
     static final class IgnoreConsume extends Consume.KeyOrValueSpec {
+
+        @Override
+        boolean isIgnore() {
+            return true;
+        }
+
+        @Override
+        public Optional<SchemaProvider> schemaProvider() {
+            return Optional.empty();
+        }
+
+        @Override
+        Deserializer<?> deserializer(KeyOrValue keyOrValue, SchemaRegistryClient schemaRegistryClient,
+                Map<String, ?> configs) {
+            return new ByteBufferDeserializer();
```

Comment:
Maybe make a `NoOpDeserializer` (without the assertion in `VoidDeserializer`)?

Reply:
`org.apache.kafka.common.serialization.ByteArrayDeserializer` may be sufficient, it just returns `data`...